bench/benchmark_cluster.py

```diff
@@ -1,8 +1,11 @@
 from thor import clusterVelocity
```

Comment:
Feel free to review this file closely, but I don't think it's very important. I just rearranged things to make it generate a lot of profiles for different parameter values (like different numbers of datapoints, `eps`, `min_samples`). It's semi-useful; a bit verbose and the result is hard to read.